roles/nvidia-dgx/vars/ubuntu.yml

```diff
@@ -180,7 +180,7 @@ PKGS_DGX1_ALL:
   - "{{ PKGS_NONVM }}"
   - "{{ PKGS_COMMON }}"
   - "{{ PKGS_DRIVER }}"
-    #- "{{ PKGS_DGX1 }}"
+# - "{{ PKGS_DGX1 }}"
```

Comment:
Why not just remove this if it is commented out? Do we expect people to use this?

Reply:
This is already merged into master. Was working off of a previous branch.